<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationsWindow</class>
 <widget class="QWidget" name="OperationsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Operations</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="1">
    <widget class="QPushButton" name="zoomButton">
     <property name="text">
      <string>Zoom</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="unzoomButton">
     <property name="text">
      <string>Unzoom</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="updateButton">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="resetButton">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="originalImageButton">
     <property name="text">
      <string>Show original image</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="imagePath"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="browseButton">
     <property name="text">
      <string>Browse</string>
     </property>
     <property name="icon">
      <iconset theme="document-open">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset theme="document-save">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="convoluateButton">
     <property name="text">
      <string>Convoluate</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="playButton">
     <property name="text">
      <string>Play simulation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <widget class="QGroupBox" name="channelsGroup">
     <property name="title">
      <string>Channels</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="grayRadio">
        <property name="text">
         <string>Gray</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rgbRadio">
        <property name="text">
         <string>RGB</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pauseButton">
     <property name="text">
      <string>Pause simulation</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="debugGroup">
     <property name="title">
      <string>Debug</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="resultsCheckBox">
        <property name="text">
         <string>Results</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="forcesCheckBox">
        <property name="text">
         <string>Forces</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="displayCheckBox">
        <property name="text">
         <string>Display</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
